### ISSUE TITLE
Top-1 accuracy on UCF101 dataset is bad?

### ISSUE THREAD

**Issue by Tramac** [dmlc/gluon-cv#1582]
I trained `i3d_nl5_resnet50_v1` on `UCF101` datasets with `Kinetics400` pretrained, the acc: `top-1=85.2%, top-5=95.4%`.
Params: `clip_len=32, input=224x224, lr=0.01, batch_size=8`.

It seems not so good, what is the possible reason?

**Comment by bryanyzhu** [comment]
Hi, maybe your lr is too large, try 0.001 instead. This is because if you use a model pretrained on K400, it is already a very good initialization. 

I didn't try `i3d_nl5_resnet50_v1 ` on UCF101 before, but I do train a `i3d_resnet50_v1` model on UCF101. The training command can be found [here](https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/action_recognition/ucf101/i3d_resnet50_v1_ucf101_kinetics400.sh) and the training log can be seen at [here](https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/action_recognition/ucf101/i3d_resnet50_v1_ucf101_kinetics400.log). The performance is good (95.4%). I suspect `i3d_nl5_resnet50_v1 `'s performance should be around 96%.

**Comment by Tramac** [comment]
Thanks for your reply, I will try it.

**Comment by Tramac** [comment]
The performance is 93.8% with `i3d_resnet50_v1` model and `lr=0.001`, the small gap may be due to the `batch_size`.

Thanks again.
